spec/Looker.3.0.oas.json

```diff
@@ -1 +0,0 @@
04":{"description":"Not Found","content":{"application/json":{"schema":{"$ref":"#/components/schemas/Error"}}}},"409":{"description":"Resource Already Exists","content":{"application/json":{"schema":{"$ref":"#/components/schemas/Error"}}}},"422":{"description":"Validation Error","content":{"application/json":{"schema":{"$ref":"#/components/schemas/ValidationError"}}}},"429":{"description":"Too Many Requests","content":{"application/json":{"schema":{"$ref":"#/components/schemas/Error"}}}}},"x-looker-status":"beta","x-looker-activity-type":"db_query"}},"/timezones":{"get":{"tags":["Config"],"operationId":"all_timezones","summary":"Get All Timezones","description":"### Get a list of timezones that Looker supports (e.g. useful for scheduling tasks).\n","responses":{"200":{"description":"Timezone","content":{"application/json":{"schema":{"type":"array","items":{"$ref":"#/components/schemas/Timezone"}}}}},"400":{"description":"Bad Request","content":{"application/json":{"schema":{"$ref":"#/components/schemas/Error"}}}},"404":{"description":"Not Found","content":{"application/json":{"schema":{"$ref":"#/components/schemas/Error"}}}}},"x-looker-status":"stable","x-looker-activity-type":"non_query"}},"/user_attributes":{"get":{"tags":["UserAttribute"],"operationId":"all_user_attributes","summary":"Get All User Attributes","description":"### Get information about all user attributes.\n","parameters":[{"name":"fields","in":"query","description":"Requested fields.","required":false,"schema":{"type":"string"}},{"name":"sorts","in":"query","description":"Fields to order the results by. Sortable fields include: name, label","required":false,"schema":{"type":"string"}}],"responses":{"200":{"description":"User Attribute","content":{"application/json":{"schema":{"type":"array","items":{"$ref":"#/components/schemas/UserAttribute"}}}}},"400":{"description":"Bad Request","content":{"application/json":{"schema":{"$ref":"#/components/schemas/Error"}}}},"404":{"description":"Not Found","content":{"application/json":{"schema":{"$ref":"#/components/schemas/Error"}}}}},"x-looker-status":"stable","x-looker-activity-type":"non_query"},"post":{"tags":["UserAttribute"],"operationId":"create_user_attribute","summary":"Create User Attribute","description":"### Create a new user attribute\n\nPermission information for a user attribute is conveyed through the `can` and `user_can_edit` fields.\nThe `user_can_edit` field indicates whether an attribute is user-editable _anywhere_ in the application.\nThe `can` field gives more granular access information, with the `set_value` child field indicating whether\nan attribute's value can be set by [Setting the User Attribute User Value](#!/User/set_user_attribute_user_value).\n\nNote: `name` and `label` fields must be unique across all user attributes in the Looker instance.\nAttempting to create a new user attribute with a name or label that duplicates an existing\nuser attribute will fail with a 422 error.\n","parameters":[{"name":"fields
```

Comment:
why did this file get deleted? I did `yarn install --force && yarn build && yarn wipe && yarn sdk`

Reply:
`yarn wipe` completely wipes that folder and we don't fetch 3.0 during codegen by default. There's a `yarn refresh` ? script that reloads the missing openapi spec for now. This is another fence-straddling step that will be obsolete when we can fetch OpenAPI specs from the server.